adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/delete-app-monitor.rst

**To deletes an existing app monitor**

> The following ``delete-app-monitor`` deletes an existing app monitor. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``delete-app-monitor`` example deletes an existing app monitor. ::
```

> 	aws rum delete-app-monitor --name AWSApp

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

The `--name` option also needs to be on it's own line.

> 

Reviewer: ```suggestion

This command produces no output

```
